service/request.go

```diff
@@ -185,6 +186,11 @@ func errorResponse(e error) (status int, ae *api.Error) {
 	status = http.StatusInternalServerError
 	ae = &api.Error{Type: "Error"}
 
+	var pe *parser.Error
+	if errors.As(e, &pe) {
+		ae.Info = map[string]int{"parse_error_offset": pe.Offset}
```

Comment:
I think this should be a struct in api- maybe `api.ParserError`?

Reply:
I'll create an issue for that.